cypress/integration/orthography.spec.js

```diff
@@ -41,5 +41,57 @@ describe('Orthography selection', function () {
         .contains('li', 'Syllabics')
         .should('have.class', 'menu-choice--selected')
     })
+
+    it('should display in syllabics given the correct cookies', function () {
+      cy.setCookie('orth', 'Cans')
+
+      // Visiting a page should be in syllabics
+      cy.visit('/about')
+      cy.contains('h1', 'ᐃᑘᐏᓇ')
+      cy.contains('.prose__heading', 'ᓀᐦᐃᔭᐍᐏᐣ')
+      cy.get('[data-cy=language-selector]')
+        .contains('Syllabics')
+    })
+
+    // XXX: This test fails in headless mode for Electron version < v6.0
+    // There was a bug with setting cookies via fetch():
+    //    https://github.com/cypress-io/cypress/issues/4433
+    // This should work in Cypress 3.5.0 and greater.
+    // As of 2020-03-05 this STILL doesn't work in CI :((((
+    it.skip('should persist my preference after a page load', function () {
+      cy.visit('/')
```

Comment:
How do you like
```javascript
if (Cypress.env('CI') === 'true'){
    this.skip()
}
```

so this beautiful test can still be run locally

Reply:
THAT'S BEAUTIFUL 🤩🤩🤩

Reply:
> I see I trapped you into another intrepid endeavor 😅
> sorry I've never actually tried to use env vars on Cypress so I have no idea of the caveats

Sneaky sneaky, train!

Yeah, so `Cypress.env()` does not work in anyway that I expect. I've reverted those commits, and I'll add the smart skipping in a new PR, since this is getting out of hand 😓